NuKeeper.Update/Process/UpdateProjectImportsCommand.cs

```diff
@@ -54,7 +55,7 @@ private static IEnumerable<string> UpdateConditionsOnProjects(Stream fileContent
 
             var imports = project.Elements(ns + "Import");
             var importsWithToolsPath = imports
-                .Where(i => i.Attributes("Project").Any(a => a.Value.Contains("$(VSToolsPath)")))
```

Comment:
In this case, the text `$(VSToolsPath)` is probably case-sensitive?

Reply:
MSBuild files are mostly case-insensitive.

Reply:
That's Ok then.